be/src/exec/vectorized/schema_scanner/schema_task_runs_scanner.cpp

```diff
@@ -187,6 +188,16 @@ Status SchemaTaskRunsScanner::fill_chunk(ChunkPtr* chunk) {
             }
             break;
         }
+        case 11: {
+            // progress
+            {
+                ColumnPtr column = (*chunk)->get_column_by_slot_id(11);
+                const std::string* str = &task_run_info.progress;
```

Comment:
progress is optional, should check __isset like error_message before accessing it